packager/packager/packager.py

```diff
@@ -115,17 +116,17 @@ def callbackFn(idx):
     # Get product count from event contents
     with tempfile.TemporaryDirectory() as td:
         print(f'Working in temporary directory: {td}')
-        outfile = write_contents_to_dssfile("/output/test.dss", basin, contents, callbackFn)
+        outfile = write_contents_to_dssfile(os.path.join(td, os.path.basename(output_key)), basin, contents, callbackFn)
 
         print('Output key is: {}'.format(output_key))
 
         # Upload the final output file to S3        
         if CONFIG.CUMULUS_MOCK_S3_UPLOAD:
             # Mock good upload to S3
             upload_success = True
-            # Copy file to tmp directory on host
+            # Copy file to output directory in the container
             # shutil.copy2 will overwrite a file if it already exists.
-            # shutil.copy2(_f["file"], "/tmp")
+            shutil.copy2(outfile, "/output/"+os.path.basename(outfile))
```

Comment:
Totally works - in the future, recommend an os.path.join() for pathnames or the new Python format strings for all other string building (f"/output/{os.path.basename(outfile)}").